pkg/kv/kvserver/logstore/logstore.go

```diff
@@ -543,6 +545,16 @@ func LoadEntries(
 
 	ents, cachedSize, hitIndex, exceededMaxBytes := eCache.Scan(ents, rangeID, lo, hi, maxBytes)
 
+	budgetFor := func(size uint64) bool {
```

Comment:
nit: `BoundAccount.Grow()` handles the `account == nil` case internally, so we don't need this helper.